<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tools</class>
 <widget class="QWidget" name="tools">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>103</width>
    <height>126</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="dockBox">
     <property name="text">
      <string>Стыковка</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="bodyPowerkBox">
     <property name="text">
      <string>Крпус</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="closeButton">
     <property name="text">
      <string>Закрыть</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
